File: 프로그래머스/unrated/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.js
function solution(strArr) {
    var answer = [];
    for(var i = 0; i < strArr.length; i++){
        if(i%2!=0){
            answer[i] = strArr[i].toUpperCase();
        }else{
            answer[i] = strArr[i].toLowerCase();
        }
    }
    return answer;
}

File: 프로그래머스/unrated/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.go
import "strings"
func solution(strArr []string) []string {
    for idx, str := range strArr{
        if idx % 2 == 0{
            strArr[idx] = strings.ToLower(str)
        }else{
            strArr[idx] = strings.ToUpper(str)
        }
    }
    return strArr
}